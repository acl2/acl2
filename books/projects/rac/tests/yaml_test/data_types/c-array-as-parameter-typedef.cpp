// RAC begin

typedef int fixed_size_arr[4];

int foo(fixed_size_arr a) { return a[2]; }

// RAC end
