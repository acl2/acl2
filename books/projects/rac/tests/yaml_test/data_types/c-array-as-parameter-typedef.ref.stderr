c-array-as-parameter-typedef.cpp:5 (23-24): Cannot pass a C array (`fixed_size_arr`) as function parameter
  5 | int foo(fixed_size_arr a) { return a[2]; }
    |                        ^
note: use array<int> instead
